src/test/java/com/browserstack/automate/ci/jenkins/local/JenkinsBrowserStackLocalTest.java

```diff
@@ -0,0 +1,24 @@
+package com.browserstack.automate.ci.jenkins.local;
+
+import org.junit.runner.RunWith;
+import com.pholser.junit.quickcheck.Property;
+import com.pholser.junit.quickcheck.runner.JUnitQuickcheck;
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+@RunWith(JUnitQuickcheck.class)
+public class JenkinsBrowserStackLocalTest {
+
+    public final String ACCESS_KEY = "dummy";
+
+    @Property
+    public void testLocalIdentifierGeneration(String buildTag) {
```

Comment:
does it cover positive and negative scenario?

Reply:
It does. It's a property-based testing library so it generates a range of inputs and tries to make the test fail.